include/bluetooth/hci_raw.h

```diff
@@ -32,6 +32,24 @@ extern "C" {
  */
 int bt_send(struct net_buf *buf);
 
+enum {
+	/* No header mode. */
```

Comment:
I was thinking on naming it PASSTHROUGH instead of NO_HEADER, that should be a little clearer by itself but I will drop a line of comment on the actual expected behavior.